### DISCUSSION TITLE
Facing issues when trying to increase Min.io storage in K8s

### DISCUSSION THREAD

**Discussion by cherrycharan** [Budibase/budibase#7549] [Help] [answered]
We are getting the below error when we try to increase the Min.io storage in k8s 
`persistentvolumeclaims "minio-data" is forbidden: only dynamically provisioned pvc can be resized and the storageclass that provisions the pvc must support resize`

**Comment by mike12345567** [comment, marked as answer]
Hi @cherrycharan - you'll need to confirm your Kubernetes version, if it is pre 1.11 persistent volumes cannot be re-sized. If you are on a newer version then you can use this feature: https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/
